src/ludamus/adapters/web/django/templatetags/tessera/dynamic_field.py

```
@@ -0,0 +1,115 @@
+                "options": [
+                    (o.value, o.label, o.value in selected, f"{input_id}_{i}")
+                    for i, o in enumerate(options)
+                ],
+                "text_value": selected[0] if selected else "",
+                "is_checked": bool(answer.value),
+                "custom_name": f"{name}_custom",
+                "custom_id": f"{input_id}_custom",
+                "custom_value": answer.custom_value,
+            },
+        )
+    )
```

Ruff (0.15.21)

[error] Use of `mark_safe` may expose cross-site scripting vulnerabilities

(S308)